### PR TITLE
Update the CSI spec support matrix

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ This information reflects the head of this branch.
 | ------------------------------------------------------------------------------------------ | ----------------------------| --------------- | -------------------- |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-snapshotter | 1.17         | v1beta1              |
 | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/snapshot-controller | 1.17     | v1beta1              |
+| [CSI Spec v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.1.0) | quay.io/k8scsi/csi-snapshotter | 1.17         | v1beta1              |
+| [CSI Spec v1.1.0](https://github.com/container-storage-interface/spec/releases/tag/v1.1.0) | quay.io/k8scsi/snapshot-controller | 1.17     | v1beta1              |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/csi-snapshotter | 1.17         | v1beta1              |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/snapshot-controller | 1.17     | v1beta1              |
```
Comment (xing-yang) [inline README.md:36]: Can you also update the image locations?
k8s.gcr.io/sig-storage/csi-snapshotter
k8s.gcr.io/sig-storage/snapshot-controller

Comment (humblec) [reply README.md]: @xing-yang done.. thanks !

**Pull request merged**